apps/web-giddh/src/app/new-inventory/component/export-inventory-master/export-inventory-master.component.html

```diff
@@ -0,0 +1,337 @@
+  <div class="modal-header">
+    <h3>Export Inventory Master</h3>
+    <button aria-hidden="true" class="close" mat-dialog-close>
+      ×
+    </button>
+  </div>
+  <mat-dialog-content class="overflow-hidden">
+    <div class="modal-body">
+      <p class="pd-l15 pb-3">
+        <span> {{ inputData?.commonLocaleData?.app_note }} : </span> 
+        {{ inputData?.localeData?.export_note }}
+      </p>
+      <form [formGroup]="exportForm" class="d-flex justify-content-between px-3 pt-2 overflow-auto">
+        <div>
+          <div class="mr-b15">
+            <label class="label-width">Stock Group Name :</label>
+            <label class="p-0 ml-2 toggle-width">
```

Comment:
why use label tag again ??? 

check and fix everywhere in this page

Reply:
done